src/invoices.ts

```diff
@@ -8,26 +8,24 @@ type InvoiceResult = {
 
 export class Invoices {
     private dispatch: Dispatch;
-    private util: Util;
     private path = 'invoices';
 
     constructor(dispatch: Dispatch){
         this.dispatch = dispatch;
-        this.util = new Util();
     }
 
     async get(documentNumber?: string) {
         const result = await this.dispatch.get(`${this.path}/${documentNumber || ''}`) as InvoiceResult;
         return result.Invoice;
     }
 
-    async getAll(filter: string) {
-        const result = await this.util.getAllPages(this.path + '?filter=' + filter, 'Invoices', this.dispatch) as InvoiceResult[];
+    async getAll(filter?: string) {
+        const result = await Util.getAllPages(this.path, 'Invoices', this.dispatch, filter) as FNInvoice[];

```

Comment:
Fix a bug - the results is array of FNInvoice[] actually, not of InvoiceResult[].